### DISCUSSION TITLE
[Resolved] "Previously Graduated"

### DISCUSSION THREAD

**Discussion by ArnavBalyan** [community/community#17792] [GitHub Education]
Hi, I tried creating a Pull Request, but it got automatically rejected because apparently I have already previously graduated. I forked the repo last year but never had my PR accepted and did not did not participate. Can someone please help me out with the issue
Edit: This has been resolved. Thank you

**Comment by juanpflores** [comment]
Hi @ArnavBalyan 👋🏼 
Thanks for opening this discussion. Glad to read it has been solved. Congratulations on graduating!

**Reply by ArnavBalyan** [reply]
Thanks
